### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards-InputsWithValidation.json]: Luminosity ratio of the focus indicator on the 'Due Date', 'Start Time' and 'What color do you want? (compact)' combo box field with respect to its adjacent background color is 1.71:1, is less than 3:1.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9217]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Tool: Color Contrast analyzer -> Launch through Accessibility Insights for Windows Tool

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'InputsWithValidation.json' control present on left navigation using tab key and activate it using enter key.
3. 'InputsWithValidation.json' adaptive card will appear on right sight of the pane.
4. Navigate to the 'Due Date', 'Start Time' and 'What color do you want? (compact)' combo box field using tab key.
5. Launch Accessibility Insights for Windows tool and open Color contrast analyzer and elect the foreground and adjacent background color of the focus indicator on the 'Due Date', 'Start Time' and 'What color do you want? (compact)' combo box field.
6. Verify that, luminosity ratio of the focus indicator on the 'Due Date', 'Start Time' and 'What color do you want? (compact)' combo box field with respect to its adjacent background color is less than minimum required ratio i.e., 3:1 or not.

**Actual Result:**
Luminosity ratio of the focus indicator on the 'Due Date', 'Start Time' and 'What color do you want? (compact)' combo box field with respect to its adjacent background color is 1.71:1, which is less than minimum required ratio i.e., 3:1.

**Note:**
1. Same issue replicates for the 'Enter the due date' combo box field present on the 'ActivityUpdate.json' card.
2. Same issue replicates for the 'Snooze for' combo box field present on the 'CalendarReminder.json' card.

**Observation:**
The focus indicator should be properly highlighted by having minimum contrast ratio 3:1 on the boundary of the 'Due Date', 'Start Time' and 'What color do you want? (compact)' control. The focus indicator should be same as the other fields such as 'Name', 'Homepage' and 'Email' etc.

**Expected Result:**
Luminosity ratio of the focus indicator on the 'Due Date', 'Start Time' and 'What color do you want? (compact)' combo box field with respect to its adjacent background color should be greater than or equal to the minimum required ratio i.e., 3:1.

**User Impact:**
Low vision users will be impacted and might not identify the currently focused element on the page if luminosity ratio of the focus indicator is less than minimum required 3:1 on the 'Due Date', 'Start Time' and 'What color do you want? (compact)' combo box field.

WCAG Reference link:
https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/b4a0f386-5cc6-4cd0-ac7a-50c861817186)

### Card JSON

```JSON
N/A
```
